### PR TITLE
Troubleshoot volume extend with "resize2fs: Permission denied" error

### PR DESCRIPTION
**Pull request by clayt0nk**
This is not the first time I run into this issue, so this I think is the best place to document it?

### CODE REVIEW

File: user/advanced-configuration/resize-disk-image.md [hunk 1]
```diff
@@ -109,7 +109,50 @@ If you have a SSD see [here][fstrim] for information on using fstrim.
 
 [fstrim]: /doc/disk-trim
 
+Troubleshooting Disk Resize
+---------------------------
 
+Generally in QubesOS a resize of a private volume is very straight-forward. 
+However, sometimes in the GUI, the resize looks like it worked, but it 
+in fact fails fails silently. If you then try the same operation in the
```
Comment (andrewdavidwong) [inline user/advanced-configuration/resize-disk-image.md:117]: Typo: "fails fails"

File: user/advanced-configuration/resize-disk-image.md [hunk 2]
```diff
@@ -109,7 +109,50 @@ If you have a SSD see [here][fstrim] for information on using fstrim.
...
+Generally in QubesOS a resize of a private volume is very straight-forward. 
+However, sometimes in the GUI, the resize looks like it worked, but it 
+in fact fails fails silently. If you then try the same operation in the 
+dom0 console with a qvm-volume extend, and it fails with an error that 
+that looks something like "resize2fs: Permission denied to resize 
+filesystem" you have probably run into something that requires more 
+extensive manual intervention.
 
+What has probably happened is that there are file system errors, which 
+means that a resize2fs will not work until fsck is run first. QubesOS
```
Comment (andrewdavidwong) [inline user/advanced-configuration/resize-disk-image.md:124]: Typo: "QubesOS" -> "Qubes OS"

**Comment by andrewdavidwong** [comment]
@unman, would you be willing to review this when you have a chance?

**Comment by andrewdavidwong** [comment]
@PROTechThor, would this be included in your troubleshooting work?

**Comment by PROTechThor** [comment]
@andrewdavidwong 

> would this be included in your troubleshooting work?

Yes. It would be on a new "Disk Troubleshooting" page, alongside https://www.qubes-os.org/doc/out-of-memory/

> If so, would you be willing to review this PR?

Sure :)

**Comment by PROTechThor** [comment]
Indeed, this issue has been encountered by many other people. 
I'm linking to this discussion here so I can reference later: [Cannot increase size of VM private storage](https://groups.google.com/g/qubes-users/c/TlCoT1jrVVY)

**Comment by andrewdavidwong** [comment]
Let me know when your review is complete, @PROTechThor.

**Comment by PROTechThor** [comment]
> Let me know when your review is complete, @PROTechThor.

@andrewdavidwong I'm not sure if there's anything else to do. I used @clayt0nk 's content at https://github.com/QubesOS/qubes-doc/pull/1059/files

**Comment by andrewdavidwong** [comment]
> > Let me know when your review is complete, @PROTechThor.
> 
> @andrewdavidwong I'm not sure if there's anything else to do. I used @clayt0nk 's content at https://github.com/QubesOS/qubes-doc/pull/1059/files

Oh, I see. I suppose we'll close this PR, then. Thank you both for your contributions.

**Pull request closed without merging**